app/views/api/application_plans/_listing.html.slim

```diff
@@ -1,20 +1,15 @@
-section class="pf-c-page__main-section pf-m-no-padding"
-  - if @plans.not_custom.size > 0
-    div id='default_plan' data=default_application_plan_data(@service, @plans)
-      = javascript_pack_tag 'default_plan_selector'
+- if @plans.not_custom.size > 0
+  div id='default_plan' data=default_application_plan_data(@service, @plans)
+    = javascript_pack_tag 'default_plan_selector'
 
-  div.pf-c-card
-    / TODO: use card body and other pf styles instead of .data class
-    / div.pf-c-card__body
-    table#plans.data
-      thead
-        tr
-          th = sortable :name, @new_plan.human_attribute_name(:name)
-          th = sortable :contracts_count, @new_plan.human_attribute_name(:contracts)
-          th = sortable :state, @new_plan.human_attribute_name(:state)
-          th.right.operations colspan="3"
-            - if can_create_plan?(@new_plan)
-              = link_to "Create #{@new_plan.model_name.human.titleize}",
-                      new_polymorphic_path([:admin, @service, @new_plan]),
-                      title: 'Create new plan', class: 'action add'
-      = render partial: 'api/plans/table_body', locals: {plans: @plans.not_custom}
+div.pf-c-card
+  / TODO: use card body and other pf styles instead of .data class
+  / div.pf-c-card__body
+  table#plans.data
+    thead
+      tr
+        th = sortable :name, @new_plan.human_attribute_name(:name)
+        th = sortable :contracts_count, @new_plan.human_attribute_name(:contracts)
+        th = sortable :state, @new_plan.human_attribute_name(:state)
+        th.right.operations colspan="3"
+    = render partial: 'api/plans/table_body', locals: {plans: @plans.not_custom}
```

Comment:
This is not inside the `<section>` anymore?

Reply:
good catch!

Reply:
Yes it is: https://github.com/3scale/porta/pull/2395/files#diff-27bada7c17b20e1a7c90053b1db234a49dbaa399881190b5d0cbe563bcdb3f50R12
But I think it belongs to the parent file. Section is the container whereas the listing (table) is the content.